win32/Lib/pywin32_testutil.py

```diff
@@ -193,6 +194,24 @@ def check_is_admin():
     return _is_admin
 
 
+# Find a test "fixture" (eg, binary test file) expected to be very close to
+# the test being run.
+def find_test_fixture(basename, extra_dir="."):
+    # look for the test file in various places
+    candidates = [
+        os.path.dirname(__file__),
+        os.path.dirname(sys.argv[0]),
+        extra_dir,
+        ".",
+    ]
+    for candidate in candidates:
+        fname = os.path.join(candidate, basename)
+        if os.path.isfile(fname):
+            return fname
+    else:
+        raise TestSkipped(f"Can't find test fixture '{fname}'")
```

Comment:
I wonder if silent test-skipping is the appropriate (default) action when file not found? The cert files at least are in the repo and always should be found when no coding error?

Current CI test runs show silent skip: "SKIPPED: 1 tests - Can't find test fixture '.\01-Intro.wav'"
Is this acceptable / intended / file erroneously missing / filename or folder or code wrong ... ?
If skipping is acceptable for some use-cases by exception, maybe optional `missing_skips=True` arg,  or None return + specific handling in the caller ... ?

(It seems in the initial PR push the CI cert tests also seemed to be skipped silently when the cert files were not found, which is probably not acceptable.)

Reply:
great question :) I opened https://github.com/mhammond/pywin32/pull/1880! This patch worked around this by installing the .cert files, which isn't necessary imo - so I've killed alot of that.

Reply:
There is still 
```
SKIPPED: 2 tests - Can't find test fixture '.\win32crypt_testcert_base64.cer'
SKIPPED: 1 tests - Can't find test fixture '.\win32crypt_testcert_bin.cer'
```
in the last logs here so far.
But after rebasing beyond d90ad11 it probably works - as happened with  '.\01-Intro.wav' ...

Reply:
Yes does work in CI run of a test rebase (conflict resolution was only in CHANGES.txt):
https://github.com/kxrob/pywin32/commit/6e3a059e9ac5c81580f0ec940894ebf9cd63bc8a